bin/docker-install.sh

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env bash
-set -eo pipefail
-
-# This file exists to make the Dockerfile easier to read.
-
-# System packages
-apt-get update
-apt-get install -y --no-install-recommends \
-    git \
-    g++ \
-    gcc \
-    curl \
-    mime-support \
-    libpcre3-dev \
-    libffi-dev \
-    libpq-dev \
-    libsasl2-dev \
-    libssl-dev \
-    libxml2-dev \
-    libxslt1-dev \
-    libz-dev
```

Comment:
These dependency installs moved over to be directly in the `Dockerfile`. When I moved the dependencies, I also removed any listed here that when gone:
- didn't cause the build to fail
- still allowed integration tests to pass

I'm not sure how robust our tests suite is though to catch regressions this might have  caused. I'm happy to add some of these dependencies back, but I would like to be able to explain with comments why we include each.

Reply:
> I'm not sure how robust our tests suite is though to catch regressions this might have caused.

I'm pretty any issue in the resulting image would be caught by the integration tests. They rely on the database, signing, etc. and each plugin is leveraged.